### PR TITLE
Bridge related pallets

### PR DESCRIPTION
**Pull request by jiguantong**
- [x] https://github.com/darwinia-network/darwinia-messages-substrate/pull/230
- [x] Darwinia > Crab
- [x] Crab > Darwinia
- [x] Update messages substrate deps

### CODE REVIEW

File: node/src/chain_spec/crab.rs [hunk 1]
```diff
@@ -217,6 +217,11 @@ pub fn config() -> ChainSpec {
 				ethereum: Default::default(),
 				evm: Default::default(),
 				base_fee: Default::default(),
+
+				// Darwinia <> Crab
```
Comment (AsceticBear) [inline node/src/chain_spec/crab.rs:222]: Suggest updating this to `S2s`,  These three crates might also be used in other bridges, like Crab <> Crab parachain.

File: node/src/chain_spec/crab.rs [hunk 2]
```diff
@@ -337,6 +342,11 @@ fn testnet_genesis(
 			},
 		},
 		base_fee: Default::default(),
+
+		// Darwinia <> Crab
```
Comment (AsceticBear) [inline node/src/chain_spec/crab.rs:348]: See the previous comment.

File: node/src/chain_spec/darwinia.rs [hunk 3]
```diff
@@ -217,6 +217,11 @@ pub fn config() -> ChainSpec {
 				ethereum: Default::default(),
 				evm: Default::default(),
 				base_fee: Default::default(),
+
+				// Darwinia <> Crab
```
Comment (AsceticBear) [inline node/src/chain_spec/darwinia.rs:222]: And this

File: node/src/chain_spec/darwinia.rs [hunk 4]
```diff
@@ -341,6 +346,11 @@ fn testnet_genesis(
 			},
 		},
 		base_fee: Default::default(),
+
+		// Darwinia <> Crab
```
Comment (AsceticBear) [inline node/src/chain_spec/darwinia.rs:352]: And this

File: runtime/crab/src/lib.rs [hunk 5]
```diff
@@ -309,6 +313,12 @@ frame_support::construct_runtime! {
 		Ethereum: pallet_ethereum = 31,
 		Evm: pallet_evm = 32,
 		BaseFee: pallet_base_fee = 33,
+
+		// Darwinia <> Crab
+		BridgeDarwiniaGrandpa: pallet_bridge_grandpa::<Instance1> = 34,
```
Comment (AsceticBear) [inline runtime/crab/src/lib.rs:318]: Please update the added pallets prefix, 34 is used by the assets pallet in the main branch.

File: runtime/darwinia/src/pallets/mod.rs [hunk 6]
```diff
@@ -100,3 +100,15 @@ mod evm;
 pub use evm::*;
 
 mod base_fee;
+
```
Comment (AsceticBear) [inline runtime/darwinia/src/pallets/mod.rs:103]: Suggest adding some notes like `#S2s stuff`.

**Review by HackFisher — APPROVED** [review]

**Comment by AurevoirXavier** [comment]
Planning review.
cc @AurevoirXavier

**Pull request merged**